src/components/Modals/Onboarding/OnboardingBasic.vue

```diff
@@ -254,6 +254,10 @@ export default defineComponent({
     'click-outside': clickOutside,
   },
   computed: {
+    entranceYears: computeEntranceYears,
```

Comment:
Is there a reason why this line isn't more similar to the gradYears() one? (line 258-260)

Reply:
`computeEntranceYears ` doesn't take a parameter, so I can just directly reference it in the Vue as `entranceYears`. since `computeGradYears` takes in a parameter, I had difficultly passing in a parameter in the Vue, so I inserted the `this.entranceYear` parameter in the definition of the function. this was the best solution I found, but if you know of a cleaner way to give the parameter for `:availableChoices="gradYears"`, let me know and I'll change it!